client/src/utils/downloads.js

```diff
@@ -12,19 +37,19 @@ export async function downloadData() {
   for (const [kind, options] of Object.entries(PLOT_AXIS_OPTIONS)) {
     /* Gets relevant fields from constants files */
     let params = [kind, ...options.y];
-    if (kind === 'm') params = params.filter((value) => value !== 'how_big');
-    const labels = params.map((param) => PLOT_AXIS_METADATA[param].label);
-    
+    if (kind === "m") params = params.filter(value => value !== "how_big");
+    const labels = params.map(param => PLOT_AXIS_METADATA[param].label);
+
     /* API request */
     const response = await axios.post(`${baseUrl}/get_object_data`, {
-      param_names: params,
+      param_names: params
     });
     const json = response.data;
```

Comment:
Some of these style changes seem wrong to me? I think the linter / style guidelines call for single quoted strings, parentheses for function shorthands, trailing commas on objects, etc so not sure what happened here. eslint was disabled for this block so I could use the for...of syntax (forEach wouldn't work) but the rest should be reverted to the usual rules :smile:

Reply:
Just ran my own linter on it and it looks good from this end. Is it a specific rule on your side @Tengelma? It should be following what is in `.eslintrc` for all of us.

Reply:
The issue is the /* eslint-disable */ at the top of this block of code -- its preventing the linter from catching the little issues so even though it seems to pass there are still some stuff that wasn't intentionally allowed. I was the one that put the thing in there because:
- Current style guidelines don't like for...of but it was 100% necessary here
- Current style guidelines don't like awaits in loops which was 100% necessary here
but it looks like this section got autoformatted to a different set of style guidelines because linting was disabled for the whole block. Probably just need to do more targeted exceptions to the linting rules.

Reply:
Ah, ya, targeted ignores are always better.

Reply:
> 
> 
> Should #96 be closed now?

Yes, I have closed it.

Reply:
Yeah, a lot of the lint fixes get overwritten I believe when I first create a branch off of dev. I am not sure why; I have resolved these lint issues with the latest push. Please re-review at your earliest convenience :)

Reply:
It looks like the exact things mentioned are fixed. So it seems like a good idea to just resolve this one and pull it in now before the meeting. Doing that now :)